<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>100</width>
      <height>630</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-color: rgb(255, 255, 255);
	color: rgb(170, 206, 56);
	border-style:solid;
	border-width:2px;
	border-radius:10px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 206, 56);
	border-color: rgb(255, 255, 255);
	border-style:solid;
	border-width:2px;
	border-radius:10px;
}
QPushButton:checked{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 206, 56);
	border-color: rgb(170, 206, 56);
	border-style:solid;
	border-width:2px;
	border-radius:10px;
}</string>
    </property>
    <widget class="QPushButton" name="btn_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>592</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>제주</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>서울</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>555</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>충북</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>518</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>충남</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>481</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>전북</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>444</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>전남</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>407</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>경북</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>경남</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>333</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>강원</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>296</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>광주</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>259</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>울산</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>222</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>부산</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>185</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>대구</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>148</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>대전</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>111</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>세종</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>74</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>인천</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>37</y>
       <width>99</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>경기</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title_lbl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>3</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>티머니 둥근바람 ExtraBold</family>
      <pointsize>33</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(101, 45, 142);</string>
    </property>
    <property name="text">
     <string>서울 관광지 내 입지 추천</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>470</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>470</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(101, 45, 142);
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:3px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>360</width>
      <height>440</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>440</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 3px;
background-color: rgb(255, 255, 255);
border-style: solid;
border-color: rgb(101, 45, 142);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>354</width>
       <height>434</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>435</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>G마켓 산스 TTF Medium</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(101, 45, 142);
color: rgb(57, 57, 57);
border-style: solid;
border-width: 3px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>160</y>
      <width>121</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>G마켓 산스 TTF Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(101, 45, 142);
border-color: rgb(101, 45, 142);
border-style: solid;
border-width:1px;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 3px;
background-color: rgb(255, 255, 255);
border-style: solid;
border-color: rgb(101, 45, 142);
border-bottom-color: rgb(255, 255, 255);
</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>3</y>
       <width>354</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Medium</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>상가종류|부동산분류|임대료|보증금|매매가|계약면적|전용면적</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ui_img/씨유로고2.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>68</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ui_img/캐릭터2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>73</y>
      <width>139</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ui_img/캐릭터3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>widget_2</zorder>
   <zorder>title_lbl</zorder>
   <zorder>widget_3</zorder>
   <zorder>scrollArea</zorder>
   <zorder>comboBox</zorder>
   <zorder>search_btn</zorder>
   <zorder>widget_4</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
